admin/spec/components/previews/solidus_admin/ui/modal/component_preview/overview.html.erb

```diff
@@ -0,0 +1,71 @@
+<div class="flex justify-center items-center h-screen">
+  <table>
+    <tr>
+      <td class="px-3 py-1 text-center">
+        <%= render component('ui/button').new(
+          scheme: :primary,
+          text: 'Modal',
+          'data-action': 'click->ui--modal#toggleModal',
+          'data-controller': 'ui--modal',
```

Comment:
We should find a way to avoid reusing the same controller for both the modal and the button opening it, but I think we can fix it later when we start using them.